qemu/tests/block_stream_simple.py

```diff
@@ -21,7 +21,7 @@ def query_status(self):
 def run(test, params, env):
     """
     block_stream_simple test:
-    1). launch block streaming job w/o set max speed
+    1). launch block streaming job w/ max speed in param "default_speed"
```

Comment:
default configure file has default_speed setting but it's configurable for user, if user drop default_speed param  by line "def default_speed",  block stream job will start by monitor command  without  speed param else block stream job will start by monitor command with speed param, so should say "with or without default speed" looks more reasonable.